build.gradle

```diff
@@ -732,6 +742,10 @@ ext.libraries = [
             exclude(group: 'org.slf4j', module: 'slf4j-api')
             force = true
         },
+        spockframework   : [
```

Comment:
I think it might be easier if you included the spock dependencies under the "test" category. One could then do: `libraries.tests` or something and have everything available that might be needed for tests.